chart/templates/postgres-minimal.yaml

```diff
@@ -32,6 +32,15 @@ spec:
     - {{ . | toYaml | nindent 6 }}
     {{- end }}
   {{- end }}
+  {{- with .Values.postgresql.tls }}
+  {{- if .secretName }}
+  tls:
+    secretName: {{ .secretName }}
+    {{- if .caSecretName }}
+    caSecretName: {{ .caSecretName }}
+    {{- end }}
+  {{- end }}
+  {{- end }}
```

Comment:
```suggestion
  {{- if .Values.postgresql.tls }}
  tls:
    {{- toYaml .Values.postgresql.tls | nindent 4 }}
  {{- end }}
```

Reply:
I think this would be a bit cleaner and would allow all the TLS options in case one needs to tweak the file names as well.